internal/iptables/iptables.go

```diff
@@ -221,7 +222,7 @@ type runner struct {
 func newInternal(exec utilexec.Interface, protocol Protocol, lockfilePath14x, lockfilePath16x string) Interface {
 	version, err := getIPTablesVersion(exec, protocol)
 	if err != nil {
-		klog.InfoS("Error checking iptables version, assuming version at least", "version", MinCheckVersion, "err", err)
+		log.Warnf(context.Background(), "Error checking iptables version, assuming version at least: %s (version=%q)", err, MinCheckVersion)
```

Comment:
We should probably pass a context here.

Reply:
Do you mean we should pass down the context from here?
https://github.com/cri-o/cri-o/blob/a5c51b8b006a6452cc596cd58bb7af30a2e78cc2/server/server.go#L395-L401

Reply:
Yes, it's the main context where the server runs.